projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad-task-definitions.php

```diff
@@ -752,16 +765,130 @@ function wpcom_add_new_page_check( $post_id, $post ) {
 }
 add_action( 'wp_insert_post', 'wpcom_add_new_page_check', 10, 3 );
 
+/**
+ * Determine if the site has an 'About' page. We do this by loading the `en` annotation for the site and theme. We then check if there is a post with the title 'About'. If there is, we get the `hs_old_id` and check to make sure a corresponding post exists in the user's locale.
+ *
+ * @return int|null The post ID of the 'About' page if it exists, null otherwise.
+ */
+function wpcom_get_site_about_page_id() {
+	$annotation = wpcom_get_theme_annotation( get_stylesheet() );
+
+	// Return null if there is no annotation or the annotation doesn't have any content.
+	if ( ! $annotation || ! isset( $annotation['content'] ) || ! is_array( $annotation['content'] ) ) {
+		return null;
+	}
+
+	// Use the annotation to build up a list of 'About' pages.
+	$headstart_about_pages = array_filter(
+		$annotation['content'],
+		function ( $post ) {
+			if ( 'page' !== $post['post_type'] ) {
+				return false;
+			}
+
+			if ( 'about' === $post['post_name'] || false !== strpos( $post['post_title'], 'About' ) ) {
+				return true;
+			}
+		}
+	);
+
+	// Return null if there are no 'About' pages.
+	if ( empty( $headstart_about_pages ) ) {
+		return null;
+	}
+
+	// Get the hs_old_ids for the 'About' pages. We'll use these to find any published pages with the same hs_old_id.
+	$headstart_about_page_hs_old_ids = array_map(
+		function ( $about_page ) {
+			return $about_page['hs_old_id'];
+		},
+		$headstart_about_pages
+	);
+
+	// Return null if there aren't any hs_old_ids to check.
+	if ( empty( $headstart_about_page_hs_old_ids ) ) {
+		return null;
+	}
+
+	$filters     = array(
+		'post_type'    => 'page',
+		'meta_key'     => '_hs_old_id',
+		'meta_compare' => 'IN',
+		'meta_value'   => $headstart_about_page_hs_old_ids,
```

Comment:
Thought: should we limit the result to one row? And might we want to specify an order, just to be super-safe?